repair/row_level.cc

```diff
@@ -845,6 +845,17 @@ class repair_meta {
     }
 
 public:
+    future<> clear_gently() noexcept {
+        return do_for_each(_peer_row_hash_sets, [] (repair_hash_set& x) {
+            x.clear();
+        }).then([this] {
+            _peer_row_hash_sets.clear();
+            return utils::clear_gently(_working_row_buf);
+        }).then([this] {
+            return utils::clear_gently(_row_buf);
+        });
+    }
```

Comment:
Please coroutinize, this is ugly.

Reply:
good idea, but I think we should do that in a follow up patch to facilitate backporting to old releases that don't support coroutines

Reply:
never mind, I'm going to reuse code from locator/token_metadata.cc that's already coroutinized,
so if and when we need a backport to a release that doesn't have coroutines, we'll deal with it there.